src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -725,6 +727,35 @@ public void Initialize()
             InitializationCompleted.Set();
         }
 
+        private void CheckSwapOrPageFileAndRaiseNotification()
+        {
+            if (PlatformDetails.RunningOnLinux)
+            {
+                if (PlatformDetails.RunningOnDocker) 
```

Comment:
On Windows-docker we are good?

Reply:
In Windows, we only check if there is a PageFile without looking at the size but I will run it on Windows Docker to make sure it works properly

Reply:
Does it work?